app/views/user_orders/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>My Orders</h1>
+<% current_user.orders.each do |order| %>
+  <article id='order-<%= order.id %>'>
+  <p>ID: <%= link_to "#{order.id}", profile_order_path(order) %></p>
+    <%= render partial: 'partials/order_info',
+      locals: {
+        id: order.id,
+        created_at: order.created_at,
+        updated_at: order.updated_at,
+        status: order.status,
+        item_orders: nil,

```

Comment:
Could the item_orders local also be omitted if it's not rendered, or must it be included?